src/recovery.rs

```diff
@@ -221,17 +252,20 @@ impl Recovery {
             }
 
             self.app_limited =
-                (self.cc.bytes_in_flight() + sent_bytes) < self.cc.cwnd();
+                (self.bytes_in_flight + sent_bytes) < self.congestion_window;
 
-            // OnPacketSentCC
-            self.cc.on_packet_sent_cc(sent_bytes, now, trace_id);
+            self.on_packet_sent_cc(sent_bytes, now);
 
             self.set_loss_detection_timer(handshake_completed);
         }
 
         trace!("{} {:?}", trace_id, self);
     }
 
+    fn on_packet_sent_cc(&mut self, sent_bytes: usize, now: Instant) {
```

Comment:
Do we need to make it as a separate function? It's a single line and pretty clear what it does so moving L266 to L257 is good enough I think.

Reply:
The idea was that common CC logic could be implemented within these methods before/after the specific CC-algorithm logic, if needed. But I don't care too much.

Reply:
Currently the cases I have is to change reno ops itself, no much necessity to add before/after the hook caller.